keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlDeliveryConfigRepository.kt

```diff
@@ -88,6 +90,51 @@ class SqlDeliveryConfigRepository(
     return deliveryConfigUIDs.size
   }
 
+  override fun delete(name: String) {
+    val deliveryConfigUid: String = jooq
+      .select(DELIVERY_CONFIG.UID)
+      .from(DELIVERY_CONFIG)
+      .where(DELIVERY_CONFIG.NAME.eq(name))
+      .fetchOne(DELIVERY_CONFIG.UID)
+
+    jooq
+      .deleteFrom(DELIVERY_CONFIG)
+      .where(DELIVERY_CONFIG.NAME.eq(name))
+      .execute()
+    jooq
+      .deleteFrom(DELIVERY_CONFIG_LAST_CHECKED)
```

Comment:
Side-question: why do we have a separate table for this in the first place?

Reply:
because that information changes a lot more frequently that the delivery configs, and it is a different type of information (cc @robfletcher ?)

Reply:
yeah, the current implementation is more efficient than continually updating all rows on `resource` and `delivery_config`.

Reply:
Got it, thanks.